README.md

```diff
@@ -94,6 +94,29 @@ spec:
 
 Accessible with the url: http://traefik.localhost/dashboard/
 
+## Upgrading
+
+```bash
+# Update repository
+helm repo update traefik/traefik
+# See current Chart & Traefik version
+helm search repo traefik/traefik
+# Upgrade Traefik
+helm upgrade traefik traefik/traefik
+```
+
+New major version indicates that there is an incompatible breaking change.
+
+### Upgrading CRDs
+
+With Helm v3, CRDs created by this chart can not be updated, cf the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions). Please read carefully release notes of Traefik before upgrading CRDs.
```

Comment:
I don't really understand what these sentences try to explain. Does this mean that user have to track traefik releases, compare releases to see if there were changes in CRDs and manually install them?

Reply:
You're right. I have updated these sentences.